### ISSUE TITLE
Wrong resize behaviour

### ISSUE THREAD

**Issue by elbachir-one** [Yazeed1s/zwm#15]
When resizing and coming to the cursor, the resizing stops at the cursor pointer.
Here is a short video:
https://streamable.com/mgkuaz

**Comment by Yazeed1s** [comment]
If many users want this to change, I will work on it

**Comment by elbachir-one** [comment]
That is cool, but that should not be the default behavior.

**Comment by Yazeed1s** [comment]
Yeah it's not expected to be the default behavior, I'll see what I can do about it later next week. If it can easily be changed I'll change it, otherwise I'll deal with it later when i have time and patience.

**Comment by deadshxll** [comment]
Been watching ZWM's development activity closely (good stuff, @Yazeed1s), haven't tested it myself, and i'm by no means knowledgeable of its source.

But I agree, this should not be default behavior.

Perhaps some logic like this should easily solve it?
```
void resize_function() {
  ...
  if (last_selected_window != current_selected_window && last_cursor_pos == current_cursor_pos) {
    force_selected_window(last_selected_window);
  }
  ...
}

// only occurs when the mouse moves
void on_mouse_move() {
  ...
  normal behavior (select the window that the mouse is over)
  ...
}
```

**Comment by Yazeed1s** [comment]
@deadshxll thanks! I think I see a solution to it, I believe I have `is_focused` flag in every node containing a window, but i am not making a use of it in the resize function. And it gets toggled on when the cursor enters a new window too, so i will have to find a way to limit this behavior and apply the changes to the resize function. I'll work on it this weekend and see how it goes.

**Comment by deadshxll** [comment]
Looking forward to it!

**Comment by Yazeed1s** [comment]
Fixed in #16 PR
